examples/embassy-usb-keyboard/src/main.rs

```diff
@@ -121,8 +121,8 @@ mod buttons {
 }
 
 #[riot_rs::config(usb)]
-fn usb_config() -> riot_rs::embassy_usb::Config<'static> {
-    let mut config = riot_rs::embassy_usb::Config::new(0xc0de, 0xcafe);
+fn usb_config() -> embassy_usb::Config<'static> {
+    let mut config = embassy_usb::Config::new(0xc0de, 0xcafe);
```

Comment:
Why is this not using the `riot_rs` reexports?

Reply:
fixed